60.md

```diff
@@ -40,6 +40,22 @@ Tags:
 * `mint` - Mint(s) this wallet uses -- there MUST be one or more mint tags.
 * `privkey` - Private key used to unlock P2PK ecash. MUST be stored encrypted in the `.content` field. **This is a different private key exclusively used for the wallet, not associated in any way to the user's Nostr private key** -- This is only used for receiving [NIP-61](61.md) nutzaps.
 
+## Wallet Backup Events
+
+A Wallet Backup Event SHOULD be created at the same time as a new Wallet Event so that future inadvertent overwrites of the `privkey` do not lead to loss of funds.
+
+```jsonc
+{
+    "kind": 375,
+    "content": nip44_encrypt([
+        [ "privkey", "hexkey" ],
+        [ "mint", "https://mint1" ],
+        [ "mint", "https://mint2" ]
+    ]),
+    "tags": []
```

Comment:
This would be even more useful if the backup event had a single character `k` (key) tag with the corresponding `pubkey` to the `privkey`. This would allow REQ filtering with `#k` to find a wallet backup for a `Proof` locked to a specific pubkey without having to load, decrypt and check every backup.


```
"tags": [
    [ "k", "<hex-pubkey>" ]
]
```

* `k`: The Nostr hex-format public key corresponding to the `privkey` field encrypted in the wallet backup.